bestie/server/test_result.go

```diff
@@ -56,6 +62,34 @@ func identifyStream(bazelBuildEvent bes.BuildEvent, streamId *build.StreamId) *b
 	return &stream
 }
 
+// Open an output file for reading.
+func openOutputFile(fileName, fileUri string) (io.ReadCloser, error) {
+	urlParts := strings.Split(fileUri, "://")
```

Comment:
Look at url.Parse: https://gobyexample.com/url-parsing

Reply:
Fixed. Using url.Parse() is a cleaner solution.